### PR TITLE
Update ios-install.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about Zero-touch that it cannot be configured on iOS devices that are enrolled without user affinity.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7756

File: microsoft-365/security/defender-endpoint/ios-install.md [hunk 1]
```diff
@@ -159,6 +159,8 @@ This step simplifies the onboarding process by setting up the VPN profile. For a
 > [!IMPORTANT]
 > Some information relates to prereleased product which may be substantially modified before it's commercially released. Microsoft makes no warranties, express or implied, with respect to the information provided here.
 
+> [!NOTE]
+> Zero-touch cannot be configured on iOS devices that are Enrolled without user affinity (user-less devices or shared devices).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/ios-install.md]:
```suggestion
> Zero-touch cannot be configured on iOS devices that are enrolled without user affinity (user-less devices or shared devices).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**